priv/emqx.schema

```diff
@@ -2259,9 +2259,11 @@ end}.
 %% key:   mnesia translational updates with per-key locks. recommended for single node setup.
 %% tab:   mnesia translational updates with table lock. recommended for multi-nodes setup.
 %% global: global lock protected updates. recommended for larger cluster.
```

Comment:
comments are no longer true after Stone's compact trie tree PR. I will rephase them in later commits